src/components/message/__internal__/message-content/message-content.component.tsx

```diff
@@ -21,6 +21,7 @@ const MessageContent = ({
       showCloseIcon={showCloseIcon}
       reduceLeftPadding={reduceLeftPadding}
       data-element="message-content"
+      data-role="message-content"
```

Comment:
**suggestion** (non-blocking): feel free to move this into a separate commit if you'd like